compiler/optimizer/ValuePropagationCommon.cpp

```diff
@@ -1539,13 +1549,16 @@ void OMR::ValuePropagation::transformArrayCopyCall(TR::Node *node)
             nextTT = nextTT->getNextTreeTop();
 
          if (trace())
-            traceMsg(comp(), "%s: n%dn %p current block_%d slowBlock block_%d newCallTree n%dn %p srcObjNode n%dn %p dstObjNode n%dn %p prevTT n%dn %p nextTT n%dn %p\n",
-               __FUNCTION__, node->getGlobalIndex(), node, _curTree->getEnclosingBlock()->getNumber(), slowBlock->getNumber(),
-               newCallTree->getNode()->getGlobalIndex(), newCallTree->getNode(),
-               srcObjNode->getGlobalIndex(), srcObjNode, dstObjNode->getGlobalIndex(), dstObjNode,
+            {
+            traceMsg(comp(), "%s: n%dn %p current block_%d slowBlock block_%d newCallTree n%dn %p prevTT n%dn %p nextTT n%dn %p\n", __FUNCTION__, node->getGlobalIndex(), node,
+               _curTree->getEnclosingBlock()->getNumber(), slowBlock->getNumber(), newCallTree->getNode()->getGlobalIndex(), newCallTree->getNode(),
                prevTT->getNode()->getGlobalIndex(), prevTT->getNode(), nextTT->getNode()->getGlobalIndex(), nextTT->getNode());
 
-         _needRuntimeTestNullRestrictedArrayCopy.add(new (trStackMemory()) TR_NeedRuntimeTestNullRestrictedArrayCopy(dstObjNode, srcObjNode,
+            traceMsg(comp(), "%s: n%dn %p srcObjNode n%dn %p #%d dstObjNode n%dn %p #%d\n", __FUNCTION__, node->getGlobalIndex(), node,
+               srcObjNode->getGlobalIndex(), srcObjNode, srcArrRefSymRef->getReferenceNumber(), dstObjNode->getGlobalIndex(), dstObjNode, dstArrRefSymRef->getReferenceNumber());
```

Comment:
This is a real nit, but is it necessary to repeat the information for `node` in this call to `traceMsg`?  It just appeared in the previous call to `traceMsg`.